data/ecosystem/integrations.yaml

```diff
@@ -42,8 +42,8 @@
 - name: Kong API Gateway
   url: http://www.konghq.com/
   docsUrl: https://docs.konghq.com/hub/kong-inc/opentelemetry/
-  components: []
-  oss: false
+  components: [Lua]
```

Comment:
Hi @svrnm! It is built-in in the Kong gateway core, should I leave that field empty in that case?

Reply:
Hm... good question, to a reader this might be confusing, because we do not provide any Lua through our project directly. Is this work in any way related to https://github.com/open-telemetry/community/issues/1276 ?

Reply:
I see what you mean, I removed this change to avoid confusion.
No, it is not related but thanks for sharing that!